jsapp/js/projects/projectsTable/projectsTableRow.module.scss

```diff
@@ -119,19 +108,82 @@ $projects-table-hover-bg: colors.$kobo-gray-98;
     &[data-field='submissions'] {
       text-overflow: clip;
     }
+
+    // To make a more forgiving checkbox interaction, make this cell
+    // much wider than it otherwise would be.
+    &[data-field='checkbox'] {
+      position: relative;
+      // create hanging pseudo-elements to expand touch targets
+      &::after,
+      &::before {
+        display: inline-block;
+        content: '';
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        z-index: 2;
+      }
+      // right: cover whitespace up to the project name link
+      &::after {
+        right: 0 - sizes.$x18;
+        width: sizes.$x18;
+      }
+      // left: cover whitespace left of project table
+      &::before {
+        left: 0 - sizes.$x32;
+        width: sizes.$x32;
+      }
+      // uncrop element
+      overflow: visible;
+      contain: inline-size style;
+
+      :global input.checkbox__input { // general appearance tweaks
+        &::after {border-radius: 1px;} // better-looking inner corners
```

Comment:
This actually makes it look worse (at least on Firefox and Chrome on retina screen), see scaled up screenshot:
<img width="860" alt="Screenshot 2023-04-19 at 22 54 17" src="https://user-images.githubusercontent.com/2521888/233198393-67bf5d19-4083-4c50-a930-6250a27bf3c7.png">

And withouth `border-radius` the issue is still present, but it's a tiny bit smaller:
<img width="880" alt="Screenshot 2023-04-19 at 23 01 57" src="https://user-images.githubusercontent.com/2521888/233199734-fa42ad53-d251-4f88-b230-f493d441cfb3.png">

Is it different on your machine?

Reply:
Yep, it was different for me, I could take screenshots. It might depend on the zoom level, too.

Reply:
I took screenshots at 100% zoom and then resized them (quickly in Preview.app so not pixel perfect).

But I think that the issue might be actually in how the Checkbox component was built (by me :P). Perhaps there is a way to set the rounded border in such way that it overflow:hidden on the blue background…

If the difference on your non-retina is bigger than on my retina, let's keep this 1px :)

Reply:
Resolved by https://github.com/kobotoolbox/kpi/pull/4442